### DISCUSSION TITLE
Option to disable captions on all videos.

### DISCUSSION THREAD

**Discussion by ZedDevStuff** [ReVanced/revanced-discussions#228] [Question]
When you start a video, sometimes captions will be on and you can't do anything about it. Its the same for video preview, sound is off and captions are on

**Comment by oSumAtrIX** [comment]
Send a video link where this happens

**Reply by v-schaefer** [reply]
At least here in Germany this video automatically activates English captions even though I've disabled closed captions in android settings:
https://www.youtube.com/watch?v=ExwqNreocpg
(youtube set to German and location to Germany)
Everytime I start that video in the android app, I have to manually disable the captions.
There are a lot of reddit discussions about that topic and no solution from google.
It would be nice to have an option to force disabling closed captions for every video

**Comment by ghost** [comment]
Android has caption settings where you can turn off captions by default. You mean the button itself?

**Reply by v-schaefer** [reply]
The Youtube app ignores that setting for some videos and activates closed captions even if it is turned off in the system settings.
